README.md

```diff
@@ -161,6 +161,16 @@ All vendor data is referenced from the **Vendor device index** file: `vendor/<ve
 endDevices:
   - device-a
   - device-b
+# The profileIDs is a distinct (preferably incremental) value for every unique profile listed in the vendor's folder.
```

Comment:
Progressive insight: there's actually no reason for it to be incremental. If vendors want to use ranges for regions because they think that's handy, they should. I know you copied this but please update.